pynumaflow/function/_dtypes.py

```diff
@@ -64,20 +58,78 @@ def items(self) -> List[Message]:
         return self._messages
 
     @classmethod
-    def as_forward_all(cls: Type[Ms], json_data: str) -> Ms:
+    def as_forward_all(cls: Type[Ms], value: bytes) -> Ms:
         msgs = cls()
-        if json_data:
-            msgs.append(Message.to_all(value=json_data.encode()))
+        if value:
+            msgs.append(Message.to_all(value=value))
         else:
             msgs.append(Message.to_drop())
         return msgs
 
-    def dumps(self, udf_content_type: str) -> str:
-        if udf_content_type == APPLICATION_JSON:
-            return json.dumps(self._messages, cls=NumaflowJSONEncoder, separators=(",", ":"))
-        elif udf_content_type == APPLICATION_MSG_PACK:
-            return msgpack.dumps(self._messages, default=msgpack_encoding)
-        raise MarshalError(udf_content_type)
+    def dumps(self) -> str:
+        return self.__str__()
```

Comment:
Magic functions are usually avoided while calling as much as possible.
```suggestion
    def dumps(self) -> str:
        return str(self)
```

Reply:
What does `magic function` mean...

Reply:
`__xxxx__` ??

Reply:
Yes in that format